CONTRIBUTING.md

```diff
@@ -49,6 +49,8 @@ the core team via a GitHub issue or [in our chat](https://www.ory.sh/chat)
 before getting started to ensure your work is consistent with ORY {{Project}}'s
 roadmap and architecture.
 
+Before you contribute to ORY's projects you need to sign our [Contributor License Agreement](https://cla-assistant.io/ory/).
```

Comment:
I would change this a bit and say:

- To merge contributions the CLA must be signed
- Briefly explain why the CLA must be signed and what it means